src/Dfe.PlanTech.AzureFunctions/Mappings/CSLinkMapper.cs

```diff
@@ -0,0 +1,9 @@
+using System.Text.Json;
+using Dfe.PlanTech.Domain.Questionnaire.Models;
+using Dfe.PlanTech.Infrastructure.Data;
+using Microsoft.Extensions.Logging;
+
+namespace Dfe.PlanTech.AzureFunctions.Mappings;
+public class CSLinkMapper(EntityRetriever retriever, EntityUpdater updater, CmsDbContext db, ILogger<CSLinkMapper> logger, JsonSerializerOptions jsonSerialiserOptions) : JsonToDbMapper<CSLinkDbEntity>(retriever, updater, logger, jsonSerialiserOptions)
```

Comment:
As well as this I think you'll need to amend `RecommendationChunkMapper` and duplicate the way it handles the referenced header except for your cslink